src/android/routes.js

```diff
@@ -0,0 +1,19 @@
+import Router from 'koa-router';
+import { getSpreadsheetValues } from '../utils/google';
+import config from '../configuration';
+
+export const router = new Router();
+
+router
+  .get('/klar', async (ctx) => {
+    const { site } = ctx.request.query;
+    const list = await getSpreadsheetValues({
+      id: config.androidSpreadsheetId,
+      range: site,
+    });
+    list.forEach((entry) => {
+      entry.focus = parseFloat(entry.focus);
+      entry.klar = parseFloat(entry.klar);
+    });
+    ctx.body = list;
```

Comment:
Probably doing the above as a `.map` operation is a little cleaner (no side effects):

```js
ctx.body = list.map((entry) => ({
   ...entry,
   focus: parseFloat(entry.focus),
   klar: parseFloat(entry.klar),
}));
```

Reply:
I'm getting some none-sense. I will leave it as is and follow up later:

```node
armenzg@armenzg-mbp backend$ yarn start
yarn run v1.7.0
$ neutrino start
✖ Building project failed
./src/android/routes.js
Module build failed: SyntaxError: Unexpected token (15:6)

  13 |     });
  14 |     ctx.body = list.map(entry => ({
> 15 |       ...entry,
     |       ^
  16 |       focus: parseFloat(entry.focus),
  17 |       klar: parseFloat(entry.klar),
  18 |     }));

BabelLoaderError: SyntaxError: Unexpected token (15:6)

  13 |     });
  14 |     ctx.body = list.map(entry => ({
> 15 |       ...entry,
     |       ^
  16 |       focus: parseFloat(entry.focus),
  17 |       klar: parseFloat(entry.klar),
  18 |     }));

    at transpile (/Users/armenzg/repos/health/backend/node_modules/babel-loader/lib/index.js:65:13)
    at /Users/armenzg/repos/health/backend/node_modules/babel-loader/lib/fs-cache.js:118:18
    at ReadFileContext.callback (/Users/armenzg/repos/health/backend/node_modules/babel-loader/lib/fs-cache.js:31:21)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:420:13)
 @ ./src/app.js 16:0-53
 @ ./src/index.js
 @ multi ./src/index (webpack)/hot/poll.js?1000
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
armenzg@armenzg-mbp backend$
/Users/armenzg/repos/health/backend/build/index.js:906
throw new Error("Module build failed: SyntaxError: Unexpected token (15:6)\n\n\u001b[0m \u001b[90m 13 | \u001b[39m    })\u001b[33m;\u001b[39m\n \u001b[90m 14 | \u001b[39m    ctx\u001b[33m.\u001b[39mbody \u001b[33m=\u001b[39m list\u001b[33m.\u001b[39mmap(entry \u001b[33m=>\u001b[39m ({\n\u001b[31m\u001b[1m>\u001b[22m\u001b[39m\u001b[90m 15 | \u001b[39m      \u001b[33m...\u001b[39mentry\u001b[33m,\u001b[39m\n \u001b[90m    | \u001b[39m      \u001b[31m\u001b[1m^\u001b[22m\u001b[39m\n \u001b[90m 16 | \u001b[39m      focus\u001b[33m:\u001b[39m parseFloat(entry\u001b[33m.\u001b[39mfocus)\u001b[33m,\u001b[39m\n \u001b[90m 17 | \u001b[39m      klar\u001b[33m:\u001b[39m parseFloat(entry\u001b[33m.\u001b[39mklar)\u001b[33m,\u001b[39m\n \u001b[90m 18 | \u001b[39m    }))\u001b[33m;\u001b[39m\u001b[0m\n");
      ^
Error: Module build failed: SyntaxError: Unexpected token (15:6)

  13 |     });
  14 |     ctx.body = list.map(entry => ({
> 15 |       ...entry,
     |       ^
  16 |       focus: parseFloat(entry.focus),
  17 |       klar: parseFloat(entry.klar),
  18 |     }));

    at Object../src/android/routes.js (/Users/armenzg/repos/health/backend/build/index.js:906:7)
    at __webpack_require__ (/Users/armenzg/repos/health/backend/build/webpack/bootstrap f4bbd016282652cb4cf1:637:1)
    at fn (/Users/armenzg/repos/health/backend/build/webpack/bootstrap f4bbd016282652cb4cf1:47:1)
    at Object../src/app.js (/Users/armenzg/repos/health/backend/build/index.js:932:75)
    at __webpack_require__ (/Users/armenzg/repos/health/backend/build/webpack/bootstrap f4bbd016282652cb4cf1:637:1)
    at fn (/Users/armenzg/repos/health/backend/build/webpack/bootstrap f4bbd016282652cb4cf1:47:1)
    at Object../src/index.js (/Users/armenzg/repos/health/backend/build/index.js:1624:63)
    at __webpack_require__ (/Users/armenzg/repos/health/backend/build/webpack/bootstrap f4bbd016282652cb4cf1:637:1)
    at fn (/Users/armenzg/repos/health/backend/build/webpack/bootstrap f4bbd016282652cb4cf1:47:1)
    at Object.0 (/Users/armenzg/repos/health/backend/build/index.js:4330:1)
```